tool/tsh/tsh.go

```diff
@@ -559,6 +567,13 @@ func Run(ctx context.Context, args []string, opts ...cliOption) error {
 		appFormatURI, appFormatCA, appFormatCert, appFormatKey, appFormatCURL, appFormatJSON, appFormatYAML),
 	).Short('f').StringVar(&cf.Format)
 
+	// Sessions.
+	sessions := app.Command("sessions", "View and control recorded sessions.").Alias("session")
+	lsSessions := sessions.Command("ls", "List recorded sessions.")
```

Comment:
Same comments as I left for tctl's new command apply here too: --limit, better descriptions, defaults.